routes/patient-routes.ts

```diff
@@ -0,0 +1,37 @@
+import { Router } from "express";
+import patientController from "../controllers/patient-controller";
+import { check } from "express-validator";
+
+const patientRoutes = Router();
+
+patientRoutes.get("/all", patientController.getAllPatients);
+
+patientRoutes.post(
+  "",
+  [
+    check("fullName").not().isEmpty().withMessage("must not be empty"),
+    check("dob").isDate().withMessage("is an invalid date format"),
+    check("gender").not().isEmpty().withMessage("must not be empty"),
+    check("phoneNumber")
+      .isMobilePhone("en-US")
+      .withMessage("is an invalid US phone number"),
+    check("address").not().isEmpty().withMessage("must not be empty"),
+    check("zipCode")
+      .isPostalCode("US")
+      .withMessage("is an invalid US postal code"),
+    check("maritalStatus")
+      .isAlphanumeric("en-US")
+      .withMessage("must only contain US alphanumeric characters")
+      .isLength({ min: 1, max: 1 })
+      .withMessage("must only contain 1 character")
```

Comment:
If you have validation from Front end, this would be mostly optional to have, especially if your backend is not expose outside your UI